openassessment/management/tests/test_collect_ora2_data.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+""" Test the collect_ora2_data management command """
+
+from mock import patch
+
+from django.core.management import call_command
+from django.core.management.base import CommandError
+
+from openassessment.test_utils import CacheResetTest
+
+
+class CollectOra2DataTest(CacheResetTest):
+    """ Test collect_ora2_data output and error conditions """
+
+    COURSE_ID = u"TɘꙅT ↄoUᴙꙅɘ"
+
+    def setUp(self):
+        super(CollectOra2DataTest, self).setUp()
+
+        self.test_header = [
+            "submission_uuid",
+            "item_id",
+            "anonymized_student_id",
+            "submitted_at",
+            "raw_answer",
+            "assessments",
+            "assessments_parts",
+            "final_score_given_at",
+            "final_score_points_earned",
+            "final_score_points_possible",
+            "feedback_options",
+            "feedback",
+        ]
+
+        self.test_row = [
```

Comment:
Is this actually test_rows?  Also, I'd have 2 rows, otherwise same test would pass even if you didn't loop.